leap.py

```diff
@@ -1,14 +1,13 @@
 # Function definition is here
 def is_leap_year( year ):
-   if year%4 == 0 and year%100 != 0:
-       print True
+   if (year%4 == 0 and year%100 != 0) or year%400 ==0:
+       print str(year) + " is a leap year"
```

Comment:
I would recommend learning Python 3 syntax. So changing `print str(year) + " is a leap year"` to `print(str(year) + " is a leap year")`. This change is actually enough to make this code run in Python 3 and still in Python 2 🎉